locale/en/docs/guides/debugging-getting-started.md

```diff
@@ -241,7 +241,7 @@ couple popular ones are listed below.
 
 The V8 Debugging Protocol is no longer maintained or documented.
 
-#### [Built-in Debugger](https://nodejs.org/dist/latest-v6.x/docs/api/debugger.html)
```

Comment:
This section is specifically referring to the Legacy Debugger though, which is different from the Inspector-based one. I don't think this link should be changed (the latest docs point to the Inspector-based debugger client while the `v6.x` docs refer to the legacy one).

cc @nodejs/diagnostics